packages/babel-plugin-transform-react-jsx/test/fixtures/regression/issue-12478-automatic/output.js

```diff
@@ -0,0 +1 @@
+const foo = /*#__PURE__*/undefined.jsx("p", {});
```

Comment:
This is a different unrelated bug: https://github.com/babel/babel/issues/12467

I'm preparing another PR for it, but it's lower priority since it's not a regression.